contents/handbook/product/releasing-as-beta.md

```diff
@@ -5,21 +5,42 @@ showTitle: true
 hideAnchor: true
 ---
 
+It's sometimes worth releasing big new features under a 'beta' label to set expectations with customers that this feature is still in active development, and might have some rough edges. We generally do this by making betas an opt-in experience for users via [the feature preview menu](http://app.posthog.com/home#panel=feature-previews).
 
-It's sometimes worth releasing big new features under a 'beta' label to set expectations with customers that this feature is still in active development, and might have some rough edges.
+If you're releasing something as a beta there are some guidelines you should follow - especially if the intent is that the beta will eventually become a full product in it's own right. 
 
-## What a beta feature should do
+## What can be released as a beta?
+A beta doesn't need to be perfect, but it should provide value to the user and have base elements of functionality. It doesn't need to be feature complete, but it should provide more than a mocked up front end. 
 
-A beta feature still needs to provide value to a customer. Mocking out the frontend without any functionality does not count as a beta feature.
+Betas do not need to be performant for high-volume users and can have big bugs, but should be clearly marked as such in the UI. 
 
-## What a beta feature doesn't have to do
+We aim not to have a beta label on features for more than two releases. 
 
-A beta feature doesn't need to have a perfect interface, does not need to be performant for high volume customers and can have big open bugs (as long as it still provides value).
+## What's required for a beta?
 
-## Time limit
+<CloudinaryImage
+  src="https://res.cloudinary.com/dmukukwp6/image/upload/goodbeta_daa2ddca2a.pngg"
+  alt="An example of a good beta"
+  className="dark:hidden"
+/>
+<CloudinaryImage
+  src="https://res.cloudinary.com/dmukukwp6/image/upload/goodbeta_dark_1dd8b2e833.png"
+  alt="An example of a good beta"
+  className="hidden dark:block"
+/> 
+<Caption>Betas should include a title, description, feedback button, and basic docs</Caption>
 
-Aim to not have a beta label on a feature for more than two releases.
+There are some best practices which all betas should follow in order to provide a minimum amount of information and usability for customers.
 
-## UX elements to show a feature is in beta
+- Betas in the feature preview menu must include a title and short description
```

Comment:
Not a big fan of the "feature preview menu must" language but I get the idea, maybe just lighten up the language a bit?

Reply:
Are there links to docs on how to do this? Or maybe a short description here?

Reply:
Added both of these - thanks!